tempesta_fw/http.c

```diff
@@ -2420,6 +2423,20 @@ tfw_http_resp_cache(TfwHttpMsg *hmresp)
 	}
 
 	/*
+	 * FSM needs TfwHttpReq to record data. It needs TfwHttpResp as well.
+	 * It will be added to the request later in tfw_http_resp_fwd(), but it's
+	 * needed now. Save previous (NULL) value for the sake of safety
```

Comment:
We just had a conversation about the place where Frang is called for responses. Actually, your initial attempt to hook `tfw_http_resp_fwd` is the most obvious. So please add a note to the comment why do we decided to hook exactly this function.